pkg/cmd/pulumi/stack_rm.go

```diff
@@ -62,33 +63,34 @@ func newStackRmCmd() *cobra.Command {
 
 			s, err := requireStack(ctx, stack, stackLoadOnly, opts)
 			if err != nil {
-				return result.FromError(err)
+				return err
 			}
 
 			// Ensure the user really wants to do this.
 			prompt := fmt.Sprintf("This will permanently remove the '%s' stack!", s.Ref())
 			if !yes && !confirmPrompt(prompt, s.Ref().String(), opts) {
 				fmt.Println("confirmation declined")
-				return result.Bail()
+				return cmdutil.ErrBail
 			}
 
 			hasResources, err := s.Remove(ctx, force)
 			if err != nil {
 				if hasResources {
-					return result.Errorf(
+					cmdutil.Diag().Errorf(diag.Message("",
 						"'%s' still has resources; removal rejected. Possible actions:\n"+
 							"- Make sure that '%[1]s' is the stack that you want to destroy\n"+
 							"- Run `pulumi destroy` to delete the resources, then run `pulumi stack rm`\n"+
-							"- Run `pulumi stack rm --force` to override this error", s.Ref())
+							"- Run `pulumi stack rm --force` to override this error"), s.Ref())
```

Comment:
Looks fine, but seeing `cmdutil.Diag().Errorf(...` for the second time makes me feel like there should really be a shorthand for using this as a print statement as this is a common pattern.